### ISSUE TITLE
VPAT — checkbox focus is not visible in all page views

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#3535]
The discussion forum forms have a checkbox labelled "Anonymous (to class)?".  This control does not have any visible marker when getting focus.  This will confuse keyboard navigation for some users.

This issue is related to issue #3146 (VPAT -- Please Do Not Trap The Tab Key).  Also see PR #3146 (VPAT -- TAB key allowed to indent in text area, ESC key navigates to next control.).

In testing the PR, `ESC` correctly sends focus to the next control (per logical control ordering), but in the discussion forum, that control happens to be a checkbox on the form labelled "Anonymous (to class)?".  There is no visible indication that this control receives focus, but it is properly interactive (press `Spacebar` and the checkbox will be ticked, as expected).

Update: It appears that all checkbox controls in all page views are affected.  This implies CSS  selecting all checkboxes to make their focus invisible.  This needs to be corrected for all checkboxes in all page views.  Also please try to make visible focus the same as all other controls (blue-ish outline).

**Comment by pbailie** [comment]
In reviewing PR #3538, I observed that the "invisible focus" is more widespread than just the discussion forum.  It appears that all checkbox controls in all page views exhibit the "invisible focus".  This implies that there is CSS, selecting all checkboxes, which makes focus invisible.

I am updating the issue description to reflect this.
